src/lib/pages/frontsite/Home/Sections/SectionsWrapper.js

```diff
@@ -70,11 +71,14 @@ export default class SectionsWrapper extends Component {
     const { sections } = this.props;
     if (!_isEmpty(sections)) {
       return (
-        <Container fluid className="fs-landing-page-section-wrapper">
-          {sections.map((Section, idx) => {
-            return <Section key={idx} className="fs-landing-page-section" />;
-          })}
-        </Container>
+        <Overridable id="SectionsWrapper.sections" sections={sections}>
```

Comment:
this should be change due to introducing react-invenio-app-ils we no longer need to keep the renderDefaultSection functions